MarvelApiClient/src/main/java/com/karumi/MarvelApiClient/model/CharacterDto.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *   Copyright (C) 2015 Karumi.
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.karumi.marvelapiclient.model;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
+
+public class CharacterDto {
+
+  @SerializedName("id") private String id;
+  @SerializedName("name") private String name;
+  @SerializedName("description") private String description;
+  @SerializedName("modified") private String modified;
+  @SerializedName("resourceURI") private String resourceUri;
+  @SerializedName("urls") private List<MarvelUrl> urls;
+  @SerializedName("thumbnail") private MarvelImage thumbnail;
+  @SerializedName("comics") private MarvelResources<ComicResource> comics;
```

Comment:
It's not the same a ComicDto and a ComicResource, comic resource it's only title and url, and not the complete date, on the api the called resources.